woocommerce-xpay-gateway.php

```
@@ -597,7 +551,10 @@ public function thankyou_page() {
          * @param bool $plain_text
          */
         public function email_instructions($order, $sent_to_admin, $plain_text = false) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$plain_text'. (undefined)

(UnusedFormalParameter)